cmd/connector/client/util.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2023, SailPoint Technologies, Inc. All rights reserved.
+package connclient
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+)
+
+const (
+	ResponseTypeOutput = "output"
+	ResponseTypeState  = "state"
+)
+
+// RawResponse represents the response format from the connector
+type RawResponse struct {
```

Comment:
nit. Should we name this struct say Connector response or so. It might read better in code below

Reply:
Yeah. Let me create another PR later to do the renaming, so that this one is not too hard to read. Thanks for the suggestion!